### PR TITLE
OEL-1292: Fields and post update changes to make oe_content compatible.

### PR DESCRIPTION
**Pull request by Maxfire**
Fields modified, tests adapted as well and post update to remove and update fields.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -40,9 +40,9 @@
         "openeuropa/oe_media": "^1.14",
         "openeuropa/oe_multilingual": "^1.9",
         "openeuropa/oe_paragraphs": "^1.12",
-        "openeuropa/oe_starter_content": "1.x-dev",
+        "openeuropa/oe_starter_content": "dev-OEL-1290",
         "openeuropa/oe_webtools": "^1.14",
-        "openeuropa/oe_whitelabel": "^1.0.0-alpha5"
+        "openeuropa/oe_whitelabel": "dev-OEL-1291"
```
Comment (drishu) [inline composer.json]: for the module it's ok, but the convention for the themes is to use a diff, you can change ..1.x for ..OEL-1291.diff, if it was discussed like this with @brummbar then I dont mind, but I'd keep to our normal way of working

**Review by drishu — CHANGES_REQUESTED** [review]

**Review by drishu — DISMISSED** [review]

File: modules/oe_showcase_page/config/install/core.entity_view_display.node.oe_showcase_page.teaser.yml [hunk 2]
```diff
@@ -19,6 +19,10 @@ content:
     third_party_settings: {  }
     region: content
 hidden:
+  oe_content_content_owner: true
```
Comment (drishu) [inline modules/oe_showcase_page/config/install/core.entity_view_display.node.oe_showcase_page.teaser.yml:22]: these will be resolved by OEL-1301

**Review by brummbar — CHANGES_REQUESTED** [review]
All looks good but let's re-export the fields and roles configuration, as they are not up to date.

**Review by brummbar — DISMISSED** [review]

**Review by abel-santos-corral — APPROVED** [review]

**Review by drishu — APPROVED** [review]

File: composer.json [hunk 3]
```diff
@@ -90,7 +90,7 @@
                 "1.x latest": "https://github.com/openeuropa/oe_bootstrap_theme/compare/1.0.0-alpha7..1.x.diff"
             },
             "openeuropa/oe_whitelabel": {
-                "feature OEL-1291": "https://github.com/openeuropa/oe_whitelabel/compare/1.0.0-alpha5..OEL-1291.diff"
+                "1.x latest": "https://github.com/openeuropa/oe_whitelabel/compare/1.0.0-alpha5..1.x.diff"
```
Comment (brummbar) [inline composer.json:93]: Let's swap with branch as mac patching is buggy when deleting code.

**Review by brummbar — CHANGES_REQUESTED** [review]
N

**Pull request merged**